nodejs/template.json

```diff
@@ -0,0 +1,73 @@
+{
+  "variables": {
+    "do_api_token": "{{env `DIGITALOCEAN_API_TOKEN`}}",
+    "image_name": "nodejs-snapshot-{{timestamp}}",
+    "node_version": "node_12.x",
+    "apt_packages": "nginx",
+    "appiication_name": "Node.js"
+  },
+  "sensitive-variables": ["do_api_token"],
+  "builders": [
+    {
+      "type": "digitalocean",
+      "api_token": "{{user `do_api_token`}}",
+      "image": "ubuntu-20-04-x64",
+      "region": "nyc3",
+      "size": "s-1vcpu-1gb",
+      "ssh_username": "root",
+      "snapshot_name": "{{user `image_name`}}"
+    }
+  ],
+  "provisioners": [
+    {
+      "type": "file",
+      "source": "common/files/var/",
```

Comment:
Eventually we will only want to include specific files. Since this is our first 1-click Packer build, we don't need to worry about copying over files we don't use.